stylesheets/SimpliGov-basic.css

```diff
@@ -62,102 +72,325 @@ h3 {
     margin-top: 5px!important;
 }
 
+
 /* add background color and padding so that it's visible above/below form */
 #workflow-wrapper {
-	/* go Dodgers (used to be: #3489ba) */
-	background-color: #223B75!important;
-	padding: 50px 0px!important;
+    /* go Dodgers (used to be: #3489ba) */
+    background-color: #223B75!important;
+    padding: 50px 0px!important;
 }
 
 /* put space between bottom of form fields and top of submit button */
 .submit-button-wrapper {
-	padding-top: 50px!important;
+    padding-top: 50px!important;
 }
 
 /* default space below submit button is 100px, too much */
 #formPartialDiv {
-	margin-bottom: 50px!important;
+    margin-bottom: 50px!important;
 }
 
-/* change color of submit button */

```

Comment:
This code is no longer necessary. It was always commented out, so it makes more sense to remove entirely, especially now that we are implementing MBTA style on the Submit button